extensions/azurecore/src/account-provider/auths/azureAuthCodeGrant.ts

```diff
@@ -82,6 +118,42 @@ export class AzureAuthCodeGrant extends AzureAuth {
 		return this.getToken(tenant, resource, postData);
 	}
 
+	private async loginWebMsal(tenant: Tenant, resource: Resource): Promise<AuthorizationCodeRequest> {
+		const callbackUri = await vscode.env.asExternalUri(vscode.Uri.parse(`${vscode.env.uriScheme}://microsoft.azurecore`));
```

Comment:
do you build webmode and test this code block?

Reply:
I don't build webmode, that still needs to be tested